test/integration/proxy-config.test.js

```diff
@@ -35,11 +40,67 @@ describe('Proxy Config', function () {
             host: 'http://proxy.com'
         });
 
-        expect(p.match.pattern).to.be('<all_urls>');
-        expect(p.getProtocols()).to.be.eql([]);
+        expect(p.match.pattern).to.be('http+https://*/*');
```

Comment:
This test will break as more protocols are accepted in ProxyConfig, make this dynamic.